qa/tasks/cephfs/test_volumes.py

```diff
@@ -142,9 +144,7 @@ def test_subvolume_create_with_invalid_data_pool_layout(self):
             if ce.exitstatus != errno.EINVAL:
                 raise
         else:
-            raise
-        # clean up
-        self._fs_cmd("subvolume", "rm", self.volname, subvolume, "--force")
```

Comment:
As you're removing this, you'll need to write a test that `fs subvolume rm --force` works as expected.